### PR TITLE
Update noaa-ofs.yaml

### PR DESCRIPTION
**Pull request by Patrick-Keown**
Adding new co-ops operational bucket

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 0e84f2905d338ffdf69c8048e92ca21e8f047c94
* Result: FAILED
* [Build Logs](https://onc2c36gc4.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=65b2f192-fbf7-4089-bc4b-ba1775b5d429%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: datasets/noaa-ofs.yaml [hunk 1]
```diff
@@ -35,6 +35,16 @@ Resources:
...
     Region: us-east-1
     Type: SNS Topic
+  - Description: CO-OPS Operational OFS Data (Historical Retention)
+    ARN: arn:aws:s3:::noaa-nos-ofs-pds
+    Region: us-east-1
+    Type: S3 Bucket
+    Explore:
+    - '[Browse Bucket](https://noaa-nos-ofs-pds.s3.amazonaws.com/index.html)'
+  - Description: New data notifications for CO-OPS OFS, only Lambda and SQS protocols allowed
+    ARN: arn:aws:sns:us-east-1:
```
Comment (zflamig) [inline datasets/noaa-ofs.yaml:45]: Oh, this is missing the rest of the arn. Do you know what it is @Patrick-Keown ?

**Comment by Patrick-Keown** [comment]
Actually can you just remove SNS for now. We need to set it up


Patrick Keown

Program Manager, NOAA Big Data Program *(Temporary Detail)*

Office of the Chief Information Officer (OCIO)

Geospatial Program Manager

Office of Coast Survey (OCS)

National Oceanic & Atmospheric Administration

(615) 319-5906 | ***@***.***

"Be sure when you step, step with care and great tact" - Dr. Seuss

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 3003b6f6d3eb5947dd6a83a63ba6f5e4107fa4a3
* Result: FAILED
* [Build Logs](https://onc2c36gc4.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=313b5c49-d3ce-4abe-8d8e-932868c41aa2%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Pull request merged**